### PR TITLE
TNO 2248 new tab for reports preview

### PR DESCRIPTION
**Pull request by kkwangsir**
In editor | admin | report

To create a new tab after clicking "generate preview" to display the report without any parent styles.

### CODE REVIEW

File: app/editor/src/features/admin/reports/utils/openPreviewInNewTab.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      <head>
+        <title>Report Preview</title>
+      </head>
+      <body>
+        <h1> ${previewContent.subject}</h1>
+        <div>${previewContent.body}</div>
+      </body>
+      </html>
+    `;
+
```
Comment (kylgarmor) [inline app/editor/src/features/admin/reports/utils/openPreviewInNewTab.ts:15]: This is a good idea, but I think it needs to be fleshed out a bit with something other than just the subject and the body.  Once this new window or tab is opened, the context of what it is will be lost.  I would suggest a bit more markup at the top of the preview page that informs that it's a preview for a specific report, and also some sort of action (button) that can be taken to close the window/tab that is clearly tied to "Close Report Preview" or something.

**Review by Fosol — COMMENTED** [review]
Am I understanding this implementation correctly?  Instead of two columns it will now open a new tab?  If so, that won't suite the design.
Is this to resolve the issue that occurs when images are pushing the width out?  If so we can edit the report template.  The template will need to work in Outlook too.

File: app/editor/src/features/admin/reports/utils/openPreviewInNewTab.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { IReportResultModel } from 'tno-core';
+
+export const openPreviewInNewTab = (previewContent: IReportResultModel) => {
+  const previewHTML = `
+      <html>
+      <head>
+        <title>Report Preview</title>
+      </head>
+      <body>
+        <h1> ${previewContent.subject}</h1>
```
Comment (kylgarmor) [inline app/editor/src/features/admin/reports/utils/openPreviewInNewTab.ts:10]: maybe just prefix this element with "Report Preview: " like this?
`<h1>Report Preview: ${previewContent.subject}</h1>`
You could do the same thing with the `<title>` as well.

**Comment by kkwangsir** [comment]
> Am I understanding this implementation correctly? Instead of two columns it will now open a new tab? If so, that won't suite the design. Is this to resolve the issue that occurs when images are pushing the width out? If so we can edit the report template. The template will need to work in Outlook too.

This is another issue TNO-2248, Bobbi wants this preview to open a separate browser tab. and be completely isolated.

**Review by Fosol — APPROVED** [review]
Nice

**Pull request merged**